content/guides/contribute/adding-new-components.mdx

```diff
@@ -1,69 +1,21 @@
 ---
 title: Adding new components
-description: Guidance to help you get your components added to Primer. The primary audience for this guidance is GitHub staff.
+description: Guidance to help build Primer Recipes. The primary audience for this guidance is GitHub staff.
 ---
 
-## On-ramp from product to Primer
+## New Components start as Primer Recipes
 
-The process starts with a new component being introduced as a custom React component in a feature team's app and becoming a Primer component once it matures and gets more usage.
+**Open Source contributors**: Please [open a discussion in Primer React](https://github.com/primer/react/discussions/new?category=ideas) if you'd like to provide feedback on existing components or suggest component additions for the library.
 
-This guidance is intended for GitHub staff members who are introducing design patterns that are not yet supported by Primer components. Contributions from Primer consumers who don't work at GitHub are welcome and encouraged, but these steps assume the reader is working on a feature team at GitHub.
+**GitHub Staff Only**: See [HubPost Guidance on creating Primer Recipes](#)
 
-### 1. Exists and is discoverable
+## Upstreaming to Primer for GitHub Staff
 
-Your component should have:
+Once you've created a Primer Recipe and it is in the adoption phase, you can nominate it for upstreaming to Primer. Primer has a strict process for adding new components to ensure they meet certain quality criteria and are accessible.
 
-- A name
-- A description
-- At least 1 Storybook story
+**Initiated by Primer:** Primer maintainers will regularly review recipes and identify candidates to upstream into the Primer libraries.
```

Comment:
There could also be new components that are added straight to Primer by Primer contributors without hitting Primer Recipes first.

Reply:
We're going to optimize for Recipes first, since that is currently, sadly, an edge case :)